jecs.d.ts

```diff
@@ -205,11 +206,17 @@ export class World {
 	parent(entity: Entity): Entity | undefined;
 
 	/**
-	 * Searches the world for entities that match specified components.
-	 * @param components The list of components to query.
-	 * @returns A Query object to iterate over results.
+	 * Checks if an entity exists in the world.
+	 * @param entity The entity to verify.
 	 */
-	query<T extends Id[]>(...components: T): Query<InferComponents<T>>;
+	contains(entity: Entity): boolean;
+
+	/**
+	 * Checks if there is any entity alive right now, regardless of generation.
```

Comment:
> `any entity`

This is a bit too vague, try Checks if an entity with the given ID is currently alive, ignoring its generation.